### PR TITLE
Figma presence & Update the IframeRegExp of DreamSub

### PR DESCRIPTION
**Pull request by diskxo**
I updated the IframeRegExp  of the metadata.json file, and edited the version of the presence

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
upload proof of the presence working

**Comment by diskxo** [comment]
> upload proof of the presence working

https://prnt.sc/v6p2l5

https://prnt.sc/v6p2sq

https://prnt.sc/v6p340

**Review by ririxichan — CHANGES_REQUESTED** [review]
Fix conflicting files

**Comment by diskxo** [comment]
Okay, I should have fixed everything

**Review by ririxichan — DISMISSED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]
bump version

**Review by ririxichan — CHANGES_REQUESTED** [review]
Well, you added DreamSub to Diziay folder, so please resolve it, and bump this version

**Review by Bas950 — CHANGES_REQUESTED** [review]
yea something is wrong with your folders mate...

**Comment by diskxo** [comment]
> yea something is wrong with your folders mate...

i'm new to github in general, and having no idea how to bump version, so i think i broke the folders ... could you remove the folder from your repository, so i can put it back as the first version?

**Review by diskxo — COMMENTED** [review]
I have reviewed the project and everything seems correct

**Comment by diskxo** [comment]
I also added another presence of the figma.com site

**Comment by ririxichan** [comment]
No,  
it doesn't...

**Comment by diskxo** [comment]
Ok, I fixed it.. -_- 

as a last thing I would like to ask you how to bump version with the repository, because I am new to github and I have no idea how to do it

File: websites/D/DreamSub/dist/metadata.json [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
+    "url": [
+        "dreamsub.stream",
+        "www.dreamsub.stream"
+    ],
+    "description": {
+        "en": "Watch all the series thanks to our archive of Anime in Streaming Sub ITA HD / FULL HD and Free Download. Find the best series ever completely free.",
+        "it": "Guarda tutte le serie grazie al nostro archivio di Anime in Streaming Sub ITA HD/FULL HD e Download gratis. Trova le migliori serie di sempre completamente gratuite."
+    },
+    "service": "DreamSub",
+    "version": "1.0.1",
```
Comment (ririxichan) [inline websites/D/DreamSub/dist/metadata.json:16]: version 1.0.2

File: websites/F/Figma/presence.ts [hunk 2]
```diff
@@ -0,0 +1,434 @@
+const presence = new Presence({
+  clientId: "768942376403075073" //Discord Client ID
+});
+
+const browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (ririxichan) [inline websites/F/Figma/presence.ts:5]: combine consts

File: websites/F/Figma/presence.ts [hunk 3]
```diff
@@ -0,0 +1,434 @@
...
+  }
+
+  if (presenceData.details == null) {
+    presence.setTrayTitle();
+    presence.setActivity();
+  } else {
+    presence.setActivity(presenceData);
+  }
+
+})
```
Comment (ririxichan) [inline websites/F/Figma/presence.ts:434]: no semicolon

**Review by ririxichan — CHANGES_REQUESTED** [review]
proofs, please change and run prettier on this because you have a spaces after ifs and not. so please fix it.

**Comment by ririxichan** [comment]
I will ask last time for proofs of working for these presences. If not we will close this ticket in ~2 days.

**Pull request merged**